src/Logging/Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <Compile Include="../../shared/*.cs" />
+    <ProjectReference Include="..\..\..\Caching\Abstractions\src\Microsoft.Extensions.Caching.Abstractions.csproj" />
```

Comment:
@anurse I had to add this reference to be able to use `ISystemClock`. This feels dirty and should be moved out of `Microsoft.Extensions.Caching.Abstractions.csproj` but not sure if 1) you agree and 2) this should be done via this PR.

Reply:
This is no go. We can't have this assembly depend on caching. Unfortunately the ISystemClock was never made to be a proper abstraction.

Reply:
Hence why I'm suggesting to move it out given we need it for this test, or an alternative approach has to be provided..

Reply:
Agreed, we can't take a reference. I think we can just rely on some level of manual testing and code review here and simplify the test (commented elsewhere) to just compare the offset of the generated timestamp to the system-reported local time zone when `UseUtcTimestamp` is `false` (even if that local time zone is UTC).

Reply:
Removed